vars/kubernetesDeployApp.groovy

```diff
@@ -0,0 +1,29 @@
+/**
+ * Make the Kubernetes namespace for an app, and deploy it with the given
+ * version.
+*/
+
+import groovy.json.JsonOutput
+
+def call(String app, String version) {
+    def namespace = "app-$app"
+    def json = JsonOutput.toJson([
+        apiVersion: 'v1',
+        kind: 'Namespace',
+        metadata: [name: namespace]
+    ])
+
+    File namespaceFile = File.createTempFile('namespace', '.tmp')
+    namespaceFile.write json
+    namespaceFile.deleteOnExit()
+
+    withKubeConfig([credentialsId: 'kubernetes-deploy-token',
+                    serverUrl: 'https://hozer-75.ocf.berkeley.edu:6443'
```

Comment:
Could this be a cname or something instead of hardcoding this hostname and having to change it later?

Reply:
Yeah I'll definitely change this out at some point. Unfortunately we still haven't named the Kubernetes masters.